src/cmd-buildextend-ec2

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python3 -u
+# An operation that mutates a build by uploading to EC2,
+# extending the meta.json with AMI information.
+
+import os,sys,json,yaml,shutil,argparse,subprocess,re,collections
+import tempfile,hashlib,gzip
+
+sys.path.insert(0, '/usr/lib/coreos-assembler')
+from cmdlib import run_verbose, write_json
+
+# Parse args and dispatch
+parser = argparse.ArgumentParser()
+parser.add_argument("--build", help="Build ID",
+                    required=True)
+parser.add_argument("--region", help="EC2 region",
+                    required=True)
+parser.add_argument("--bucket", help="S3 Bucket",
+                    required=True)
+args = parser.parse_args()
+
+with open('src/config/manifest.yaml') as f:
+    manifest = yaml.safe_load(f)
```

Comment:
as part of the rework i'm doing soon I'm going to try to document all the fields in the manifest.yaml and what they're used for.